src/components/Character.jsx

```diff
@@ -1,17 +1,23 @@
 import React from 'react';
+import { Draggable } from 'react-beautiful-dnd';
 
-const Character = (props) => {
-  // render individual character list with stats
-
-  const { name, lineCount, speakCount } = props.characterData;
+// Individual draggable character card with character data.
+const Character = ({character, index}) => {
+  // text is set as a string so the new line isn't displayed
```

Comment:
I'm not sure I understand this comment and the \n below. What's going on? 
If the `\n` isn't displayed, why include it in the string?

Reply:
You're right that this is confusing. I had been trying to have it display as two lines but if I put the string directly in the p tag with the new line it would show the \n text itself. Once I took half a sec to think about it, I realized I could just have two p elements and that solves it.